style.css

```diff
@@ -87,6 +87,29 @@ a {
 	font-family: var( --font-body );
 }
 
+.color-accent,
```

Comment:
Are these accent color styles meant to be bundled in this PR?

Reply:
Yeah, I noticed them missing when I was looking at author pages. Wasn't worth starting a new branch imo.